src/main/java/com/strandls/esmodule/controllers/ESController.java

```diff
@@ -483,6 +483,8 @@ public MapResponse search(@PathParam("index") String index, @PathParam("type") S
 			@QueryParam("geoAggegationPrecision") Integer geoAggegationPrecision,
 			@QueryParam("onlyFilteredAggregation") Boolean onlyFilteredAggregation,
 			@QueryParam("termsAggregationField") String termsAggregationField,
+			@QueryParam("geoFilterField") String geoShapeFilterField,
```

Comment:
Please dont modify exisitng api defination , else the other module using it will fail, 
And if its neccesary change then please modify in the other place accordingly